src/components/app/ListOfPublishedProfiles.css

```diff
@@ -58,6 +58,14 @@
   background: var(--grey-90-a05);
 }
 
+/**
+ * Overcome the selector specificity of the focus by including the "a"
+ */
+a.publishedProfilesLink:focus {
```

Comment:
I think this is the left border from https://github.com/firefox-devtools/profiler/blob/e1d2bdf80544900ac8b7a7e68faae2e7fa2d1aa4/src/components/app/ListOfPublishedProfiles.css#L61-L64

It's enabled with the :focus-within, the goal is that we see what line is currently used when we're focusing the delete button. So maybe we can instead make it more "focused" (pun intended) with:
```css
.publishedProfilesListItem .buttonWithPanelButton:focus
```
or
```css
.publishedProfilesListItem > .publishedProfilesActionButtons:focus-within
```
(keeping the `.publishedProfilesListItem_ConfirmDialogIsOpen`)

With this change, the left blue border wouldn't appear when focusing the link, it would appear when focusing the buttons only.

Alternative: try and move this border further left, so that they're not confused anymore.